src/core/directives/google-map.ts

```diff
@@ -166,35 +161,46 @@ export class SebmGoogleMap implements OnChanges, OnInit, OnDestroy {
   usePanning: boolean = false;
 
   /**
+   * Sets the viewport to contain the given bounds.
+   */
+  fitBounds: LatLngBoundsLiteral|LatLngBounds = null;
+
+  /**
+   * The enabled/disabled state of the Zoom control.
+   */
+  zoomControl: boolean | PositionOption = true;
+
+  /**
    * The initial enabled/disabled state of the Street View Pegman control.
    * This control is part of the default UI, and should be set to false when displaying a map type
    * on which the Street View road overlay should not appear (e.g. a non-Earth map type).
    */
-  streetViewControl: boolean = true;
+  streetViewControl: boolean | PositionOption = true;
 
   /**
-   * Sets the viewport to contain the given bounds.
+   * The initial enabled/disabled state of the Scale control. This is disabled by default.
    */
-  fitBounds: LatLngBoundsLiteral|LatLngBounds = null;
+  scaleControl: boolean | PositionOption = false;
```

Comment:
I think we should stay in line with the google maps options attributes. Therefore, scaleControl (and the other *Control attributes) can only be numbers. 
IMO in addition to that we also need new *Options Attributes (for example RotateControlOptions).